kolibri/core/assets/test/vue.content-renderer.js

```diff
@@ -0,0 +1,505 @@
+/* eslint-env mocha */
+// The following two rules are disabled so that we can use anonymous functions with mocha
+// This allows the test instance to be properly referenced with `this`
+/* eslint prefer-arrow-callback: "off", func-names: "off" */
+
+'use strict';
+
+const Vue = require('vue-test');
+const contentRenderer = require('../src/vue/content-renderer');
+const ContentRendererComponent = Vue.extend(contentRenderer);
```

Comment:
Create an actual Vue component so that we can call the `new` keyword on it to instantiate.